js/gallery/static/Gallery.svelte

```diff
@@ -131,7 +131,7 @@
 			if (selected_image !== null) {
 				dispatch("select", {
 					index: selected_image,
-					value: _value?.[selected_image][1]
+					value: [_value?.[selected_image][0].data, _value?.[selected_image][1]]
```

Comment:
Before, the value of the selection was just the caption, which was usually `None`. This changes it to be [image_data, caption], where the image_data is usually a filepath/url.